packages/mco/components/mco-dashboard/data-policy/alert-card/AlertItem.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import './alert-item.scss';
+import { Alert } from '@openshift-console/dynamic-plugin-sdk';
+import { Timestamp } from '../../../../../shared/details-page/timestamp';
+import {
+  RedExclamationCircleIcon,
+  YellowExclamationTriangleIcon,
+} from '../../../../../shared/status/icons';
+import { getAlertSeverity } from '../../../../../shared/utils';
```

Comment:
```suggestion
import { getAlertSeverity } from '@odf/shared/utils';
```
Do this for all shared relative imports.